source/lib/cdk-textract-stack.ts

```diff
@@ -91,7 +105,9 @@ export class CdkTextractStack extends cdk.Stack {
         bucketName: this.resourceName("sample-s3-bucket"),
         versioned: false,
         cors: [corsRule],
-        encryption: BucketEncryption.S3_MANAGED
+        encryption: BucketEncryption.S3_MANAGED,
+        serverAccessLogsBucket: logsS3Bucket,
+        serverAccessLogsPrefix: "samplesS3Logs"
```

Comment:
Same comment on prefix name as for document-s3-bucket

Reply:
Updated prefix